src/controller/RacingGameController.js

```diff
@@ -0,0 +1,72 @@
+const RacingGame = require('../model/RacingGame');
+const inputView = require('../view/inputView');
+const outputView = require('../view/outputView');
+const validate = require('../utils/validation');
+const { SIGN, ERROR } = require('../utils/constants');
+
+class RacingGameController {
+  #inputView;
+  #outputView;
+  constructor() {
+    this.#inputView = inputView;
+    this.#outputView = outputView;
+    this.initiate();
+  }
+
+  initiate() {
+    this.#readCarNames();
+  }
+
+  #readCarNames() {
+    this.#inputView.readUserInputNames((input) => {
+      const names = input.split(SIGN.comma);
+      this.#handleCarNamesException(names);
+    });
+  }
+
+  #readTrialCounts(game) {
+    this.#inputView.readUserInputTrialCounts((input) => {
+      this.#handleTrialCountsException(input, game);
+    });
+  }
+
+  #handleCarNamesException(names) {
+    try {
+      this.#validateCarNames(names);
+      const racingGame = new RacingGame(names);
+      this.#readTrialCounts(racingGame);
+    } catch (error) {
+      this.#outputView.printError(ERROR.userInputNameInvalid);
+      this.#readCarNames();
+    }
+  }
+
+  #validateCarNames(names) {
+    for (const item of names) {
+      validate.userInputName(item);
+    }
+  }
```

Comment:
```suggestion
  #validateCarNames(names) {
    names.forEach(name => {
      validate.userInputName(name)
    })
  }
```
코멘트 달면서 느낀건데 forEach랑 for of 중 뭘 사용 해야할 지 나도 기준이 명확하지 않네
성능은 크롬에서 for of가 더 좋다고 하는데 유의미한 차이는 아니야

개인적으로 나는 forEach는 반드시 모든 원소를 순회할 때 쓰는 편이고 for of는 중간에 break를 해야한다거나 await을 해야 할 때 쓰는 편이야
for of가 더 가능한게 많고 자유로워서 무조건 낫다고 생각할 수 있는데 어떤 상황에서는 제약이 있는 편이 오히려 발생할 수 있는 상황들을 제어해줘서 가독성에 도움이 될 수 있어. 기준을 한번 세워봐